magstock/tests/__init__.py

```diff
@@ -0,0 +1,11 @@
+import pytest
```

Comment:
These are the Sideboard-provided example unit tests.  I should probably remove these and/or write some real ones :)